big-red-shuttle/GPS.swift

```diff
@@ -107,15 +116,15 @@ import SwiftyJSON
     }
     
     func notifyObserversGPSBeganLogging() {
-        let _ = self.observers.map({ $0?.gpsBeganLogging?(gps: self) })
+        let _ = self.observers.map({ $0?.gpsStartedLogging?(gps: self) })
```

Comment:
Scratch that, they actually left in forEach. I thought they removed it but maybe I had only read that on the mailing list.
```
[0,1,2].forEach { (x) in
    print(x)
}
```